configuration-wire/precomputed-obfuscated.json

```diff
@@ -0,0 +1,21 @@
+{
+  "version": 1,
+  "precomputed": {
+    "subjectKey": "test-subject-key",
+    "subjectAttributes": {
+      "device": "iPhone",
+      "country": "USA"
+    },
+    "fetchedAt": "2024-11-18T14:23:39.456Z",
+    "response": {
```

Comment:
In configuration wire format, the whole `response` is a string (not `flags`). (This is so that SDKs could wrap server responses into this format without parsing them.)

Here's an example configuration wire:
```json
{
  "version": 1,
  "precomputed": {
    "subjectKey": "test-subject-1",
    "subjectAttributes": {},
    "response": "{\"obfuscated\":true,\"format\":\"PRECOMPUTED\",\"salt\":\"l0keO0Np+Vt4UbknemwiNw==\",\"createdAt\":\"2024-12-10T08:23:44.887464Z\",\"environment\":{\"name\":\"Test\"},\"flags\":{\"c79984d49220ab9ac5e3c00461582411\":{\"variationType\":\"INTEGER\",\"variationValue\":\"MQ==\",\"doLog\":true,\"allocationKey\":\"NTAvNTAgc3BsaXQ=\",\"variationKey\":\"b25l\",\"extraLogging\":{}},\"55fef4d1ec28513ab2869de4bc656e32\":{\"variationType\":\"BOOLEAN\",\"variationValue\":\"ZmFsc2U=\",\"doLog\":true,\"allocationKey\":\"b2ZmLWZvci1hbGw=\",\"variationKey\":\"b2Zm\",\"extraLogging\":{}},\"c884de61c22e12e8a2ec98ad0d7cebc2\":{\"variationType\":\"STRING\",\"variationValue\":\"Y3VycmVudA==\",\"doLog\":true,\"allocationKey\":\"Y3VycmVudC12ZXJzaW9ucw==\",\"variationKey\":\"Y3VycmVudA==\",\"extraLogging\":{}},\"b1368e40ef5c874631d7aebe0704eb91\":{\"variationType\":\"STRING\",\"variationValue\":\"b2xk\",\"doLog\":true,\"allocationKey\":\"bnVsbC1vcGVyYXRvcg==\",\"variationKey\":\"b2xk\",\"extraLogging\":{}},\"15f4f3e2274975e327aaea7b2e0373e4\":{\"variationType\":\"JSON\",\"variationValue\":\"eyAiaW50ZWdlciI6IDIsICJzdHJpbmciOiAidHdvIiwgImZsb2F0IjogMi4wIH0=\",\"doLog\":true,\"allocationKey\":\"NTAvNTAgc3BsaXQ=\",\"variationKey\":\"dHdv\",\"extraLogging\":{}},\"d6bef173e6de590815fad0fe4a974450\":{\"variationType\":\"NUMERIC\",\"variationValue\":\"My4xNDE1OTI2\",\"doLog\":true,\"allocationKey\":\"cm9sbG91dA==\",\"variationKey\":\"cGk=\",\"extraLogging\":{}}}}"
  }
}
```

Reply:
Good catch! Updated